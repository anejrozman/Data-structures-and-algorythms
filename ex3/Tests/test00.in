5,4,2
3.8031099814139555,3.644517942977693,4.92320769575414,3.0003846259799665,7.558593976449384
2,0,130.488
1,3,451.602
4,1,947.615
0,4,222.185
