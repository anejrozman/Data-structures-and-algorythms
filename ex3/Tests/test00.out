3,84.6822
2,10.935
